gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/initialize.flow.ts

```diff
@@ -3,45 +3,101 @@ import { TransactionResponse, waitExecute } from '@chainlink/gauntlet-solana'
 
   constructor(flags, args) {
     super(flags, args, waitExecute, makeAbstractCommand)
 
     this.stepIds = {
-      BILLING_ACCESS_CONTROLLER: 1,
-      REQUEST_ACCESS_CONTROLLER: 2,
-      OCR_2: 3,
+      OCR_2: 1,
     }
 
     this.flow = [
       {
-        name: 'Initialize Billing AC',
-        command: InitializeAC,
-        id: this.stepIds.BILLING_ACCESS_CONTROLLER,
+        name: 'Initialize OCR 2',
+        command: Initialize,
+        flags: {
+          billingAccessController: process.env.BILLING_ACCESS_CONTROLLER || this.flags.billingAccessController,
+          requesterAccessController: process.env.REQUESTER_ACCESS_CONTROLLER || this.flags.requesterAccessController,
+          link: process.env.LINK || this.flags.link,
+        },
+        id: this.stepIds.OCR_2,
       },
       {
-        name: 'Initialize Request AC',
-        command: InitializeAC,
-        id: this.stepIds.REQUEST_ACCESS_CONTROLLER,
+        name: 'Change RDD',
+        exec: this.showRddInstructions,
       },
       {
-        name: 'Initialize OCR 2',
-        command: Initialize,
+        name: 'Begin Offchain Config',
+        command: BeginOffchainConfig,
         flags: {
-          billingAccessController: FlowCommand.ID.contract(this.stepIds.BILLING_ACCESS_CONTROLLER),
-          requesterAccessController: FlowCommand.ID.contract(this.stepIds.REQUEST_ACCESS_CONTROLLER),
-          link: this.flags.link || process.env.LINK_ADDRESS,
-          decimals: Number(this.flags.decimals),
+          state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+          version: this.flags.version || 1,
+        },
+      },
+      {
+        name: 'Write Offchain Config',
+        command: WriteOffchainConfig,
+        flags: {
+          state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+        },
+      },
+      {
+        name: 'Commit Offchain Config',
+        command: CommitOffchainConfig,
+        flags: {
+          state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+        },
+      },
+      {
+        name: 'Set Config',
+        command: SetConfig,
+        flags: {
+          state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+        },
+      },
+      {
+        name: 'Set Payees',
+        command: SetPayees,
+        flags: {
+          state: FlowCommand.ID.contract(this.stepIds.OCR_2),
+          link: process.env.LINK || this.flags.link,
+        },
+      },
+      {
+        name: 'Set Billing',
+        command: SetBilling,
+        flags: {
+          state: FlowCommand.ID.contract(this.stepIds.OCR_2),
         },
-        id: this.stepIds.OCR_2,
       },
     ]
   }
+
+  showRddInstructions = async () => {
+    logger.info(
+      `
+        Change the RDD ID with the new state address: 
+          - State Address: ${this.getReportStepDataById(FlowCommand.ID.contract(this.stepIds.OCR_2))}
+        Include the transmissions account into the contract information:
+          - Transmissions Account: ${this.getReportStepDataById(
```

Comment:
These are the same thing, right? Let's label them the same, either "Address" or 'Account"